.github/workflows/ci.yml

```diff
@@ -14,10 +14,10 @@ env:
   TOX_TESTENV_PASSENV: FORCE_COLOR
   PIP_DISABLE_PIP_VERSION_CHECK: "1"
   PIP_NO_PYTHON_VERSION_WARNING: "1"
-  # Keep in sync with .pre-commit-config.yaml/default_language_version/python.
-  PYTHON_LATEST: "3.11"
```

Comment:
@hynek FYI, there's a new context in GHA — `${{ vars }}`. You can control it from a centralized location like repo settings, it could be accessed as `${{ vars.PYTHON_LATEST }}` then.

Reply:
Then, the contributing guide should hint the contributors how to ignore this invasive config in Git, I think.